tox.ini

```diff
@@ -1,8 +1,9 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,pypy,pypy3
+envlist = py27,py35,py36,py37,py38,pypy,pypy3
 
 [testenv]
 deps =
     setuptools >= 17.1
+    mock <= 3.0.5
```

Comment:
I'm curious why the pin was necessary. Could this instead be tox-environment specific, or use an dependency qualifier (`thing <= version; python_version == "2.7"`)?

Reply:
Python 2.7 build broke because it installed the latest Python 3 only Mock version.

My intent was just to make the test pass.

@icemac is preparing a tool which unifies all Zope repos - and possibly will also take care of these kind of problems.

If you are ok, I'd like to merge it anyway for now. On the other hand... yeah, you are right... I push an update.

Reply:
> Python 2.7 build broke because it installed the latest Python 3 only Mock version.

Wow, that means it used a very ancient pip that doesn't understand python_requires.

Now, tox uses pip it gets inside virtualenvs it creates, so which pip version you get depends on which virtualenv version you have (virtualenv bundles a copy of the pip wheel for offline installs).  I used to have this "old pip" problem on my Appveyor builds, and I fixed them by doing a `pip install -U virtualenv` before running `tox`.

Reply:
Ok, you can do this for CI.. but what about running tox locally? huh?

Reply:
I think it's reasonable to assume/require developers don't have ancient versions of build tools (pip, setuptools, wheel, tox, virtualenv) in their environment.

Support for python_requires was added in pip 9, released in 2016.